docs/src/main/asciidoc/native-and-ssl.adoc

```diff
@@ -219,6 +220,23 @@ EXPOSE 8080
 CMD ["./application", "-Dquarkus.http.host=0.0.0.0", "-Djava.library.path=/work/lib", "-Djavax.net.ssl.trustStore=/work/cacerts"]
 ----
 
+==== Java 11
+[source, subs=attributes+]
+----
+FROM quay.io/quarkus/ubi-quarkus-native-image:{graalvm-version}-java11 as nativebuilder
+RUN mkdir -p /tmp/ssl-libs/lib \
+  && cp /opt/graalvm/lib/security/cacerts /tmp/ssl-libs \
+  && cp /opt/graalvm/lib/libsunec.so /tmp/ssl-libs/lib/
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal
+WORKDIR /work/
+COPY --from=nativebuilder /tmp/ssl-libs/ /work/
+COPY build/*-runner /work/application
```

Comment:
Let's try and keep this the same as the Java 8 example. I assume you were using Gradle when you tried this :)

Reply:
Thanks for the hint! I fixed it. 

Btw, is there a "selection" thing for the docs to provide code for different setups and let the user easily switch between them? E.g. Java/Kotlin with Maven/Gradle? I imagine something like you can see on the [Android Developers pages with Java/Kotlin](https://developer.android.com/topic/libraries/architecture/livedata#kotlin). I think this could really help the docs in the future when you make Quarkus fully work with the combinations mentioned above.

Reply:
We don't have that yet, but that's something we'll definitely need to address in the future